scope.py

```diff
@@ -599,147 +609,207 @@ def train(
         run_eagerly = kwargs.get("run_eagerly", False)
         pre_trained_model = kwargs.get("pre_trained_model")
         save = kwargs.get("save", False)
+        plot = kwargs.get("plot", False)
         weights_only = kwargs.get("weights_only", False)
 
+        # xgb-specific arguments
```

Comment:
Can we add some code / document these?

Reply:
I'll add some comments in the code describing these parameters in more detail.